### DISCUSSION TITLE
React Hook form Testing using enzyme

### DISCUSSION THREAD

**Discussion by subrato-pattanaik** [react-hook-form/react-hook-form#3227] [Q&A] [answered]
I have posted a question on the spectrum.
https://spectrum.chat/react-hook-form/help/react-hook-form-testing-using-enzyme~57ea6f9c-bcaf-4030-ab60-516bb74e6523

Need some help

**Comment by subrato-pattanaik** [comment]
Please I have also posted in the StackOverflow link. The link has been deleted for good reason.

**Reply by bluebill1049** [reply]
i have no idea, others help plz :)

**Reply by subrato-pattanaik** [reply]
Thanks for the reply!!! :)

**Comment by subrato-pattanaik** [comment, marked as answer]
There was something wrong with my javascript file not in the testing file. That was my bad. Sorry !!
